### PR TITLE
ilmbase 2.2.0

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

File: Library/Formula/ilmbase.rb [hunk 1]
```diff
@@ -1,10 +1,8 @@
-require 'formula'
-
 class Ilmbase < Formula
-  homepage 'http://www.openexr.com/'
-  url 'http://download.savannah.gnu.org/releases/openexr/ilmbase-2.1.0.tar.gz'
-  mirror 'http://download-mirror.savannah.gnu.org/releases/openexr/ilmbase-2.1.0.tar.gz'
-  sha1 '306d76e7a2ac619c2f641f54b59dd95576525192'
+  homepage "http://www.openexr.com/"
+  url "http://download.savannah.nongnu.org/releases/openexr/ilmbase-2.2.0.tar.gz"
```
Comment (tdsmith) [inline Library/Formula/ilmbase.rb]: Do I want to ask what `nongnu` is about?

Comment (dunn) [inline Library/Formula/ilmbase.rb]: Not 100 percent sure? But it looks just like a mirror for Savannah stuff.

Comment (tdsmith) [inline Library/Formula/ilmbase.rb]: Oh, it's on the front page:

> Savannah aims to be a central point for development, maintenance and distribution of official GNU software. In addition, for projects that support free software but are not part of GNU, we provide savannah.nongnu.org.

**Comment by tdsmith** [comment]
Thanks!

**Pull request closed without merging**